src/FileFormats/format_flatbuffer_opl3.cpp

```diff
@@ -0,0 +1,206 @@
+/*
+ * OPL Bank Editor by Wohlstand, a free tool for music bank editing
+ * Copyright (c) 2016-2018 Vitaly Novichkov <admin@wohlnet.ru>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "format_flatbuffer_opl3.h"
+#include "../common.h"
+#include "Opl3Bank_generated.h"
+
+bool FlatbufferOpl3::detect(const QString &, char *magic)
+{
+    return true;
+}
```

Comment:
Detect is must to identify file format. That means to avoid possible reading of another file format by thins parser. If is impossible to detect by magic number, then by filename extension or by file sizes, etc.

Reply:
I don't think it's possible, so it will be using the file extension